src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -75,10 +77,15 @@ interface IManageApplicationProps {
 class ManageApplicationContainer extends React.Component<
   IManageApplicationProps,
   IManageApplicationState
-> {
+  > {
   constructor(props: IManageApplicationProps) {
     super(props);
     this.state = {
+      settings: {
+        openTime: new Date().toString(),
```

Comment:
nit: why store as string then covert back to Date later?

Reply:
Type issues. These are string types.

Reply:
Ah, makes sense